diff --git a/pango/pango-layout.c b/pango/pango-layout.c
index c4a3141c..6bd3f63d 100644
--- a/pango/pango-layout.c
+++ b/pango/pango-layout.c
@@ -6806,7 +6806,7 @@ apply_baseline_shift (PangoLayoutLine *line,
           if (item->analysis.flags & PANGO_ANALYSIS_FLAG_CENTERED_BASELINE)
             baseline_tag = HB_OT_LAYOUT_BASELINE_TAG_IDEO_EMBOX_CENTRAL;
           else
-            baseline_tag = hb_ot_layout_get_horizontal_baseline_tag_for_script (script);
+            baseline_tag = HB_OT_LAYOUT_BASELINE_TAG_ROMAN;
           hb_ot_layout_get_baseline_with_fallback (hb_font,
                                                    baseline_tag,
                                                    direction,
